Modules/HRV_DFA/DFA.cpp

```diff
@@ -0,0 +1,231 @@
+#include "DFA.h"
+
+
+void Polyfit::createNarray()
+{
+    int m_Size = m_End-m_Start+1;
+    m_N.resize(m_Size);
+    std::iota(m_N.begin(), m_N.end(), m_Start);
+}
+
+void Polyfit::readInput()
+{
+    // this only for debugging
+    // delete before merge
+    std::ifstream ifile ("C://Users//Natalia//Desktop//dadm projekt//ECG-Analyzer//Modules//HRV_DFA//nsr001.txt");
+    double m_Num = 0.0;
+
+    std::vector <double> m_Data;
+    
+    if (!ifile.is_open()) {
+            std::cerr << "There was a problem opening the input file!\n";
+            exit(1);//exit or do additional error checking
+        }
+
+    while (ifile >> m_Num) {
+            m_Data.push_back(m_Num);
+        }
+
+    int m_Length = int(m_Data.size());
+
+    // for proper application
+    // uncomment before merge
```

Comment:
Proszę przetestuj działanie modułu z tym kodem. Jeżeli on coś wysypie to nie chcemy go mergować

Reply:
Nie mam danych do tego modułu. W jaki sposób dostarczyć przykładowe dane?

Reply:
done